QUEST.tsv

```diff
@@ -102,7 +102,7 @@ QUEST_20150317_000101	There are monsters beyond this point. Be careful.
 QUEST_20150317_000102	These woods aren't exactly peaceful, but it's better than other places. {nl}There are forests that have nothing but spirits roaming about.
 QUEST_20150317_000103	I look for people like you, Revelators, the ones who have seen the Goddess in their dreams.
 QUEST_20150317_000104	From the beginning, I thought of the Revelators' visions as madness. {nl}The Goddesses have been gone for a long time after all.
-QUEST_20150317_000105	Haven't you heard about it? They've not answered our players since Medis Diena four years ago.
+QUEST_20150317_000105	$Since that day four years ago, isn't it true that the goddesses still doesn't respond to our prayers?
```

Comment:
*Since that day four years ago* -> *Since Medis Diena four years ago* - The name of the event is described in the Korean counterpart.